sw/device/lib/dif/autogen/dif_mbx_autogen.c

```diff
@@ -0,0 +1,263 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// THIS FILE HAS BEEN GENERATED, DO NOT EDIT MANUALLY. COMMAND:
+// util/make_new_dif.py --mode=regen --only=autogen
+
+#include "sw/ip/mbx/dif/autogen/dif_mbx_autogen.h"
```

Comment:
I think this was changed by mistake? This uses the path in integrated_dev

Reply:
Something went south here during the pull/rebase (2 machines). I'll take a look.